lib/std/json.zig

```diff
@@ -3257,3 +3257,22 @@ test "stringify struct with custom stringifier" {
 test "stringify vector" {
     try teststringify("[1,1]", @splat(2, @as(u32, 1)), StringifyOptions{});
 }
+
+// Same as `stringify` but accepts an Allocator and stores result in dynamically allocated memory instead of using a Writer.
+// Caller owns returned memory.
+pub fn stringifyAlloc(allocator: *std.mem.Allocator, options: StringifyOptions, value: anytype) ![]const u8 {
+    var list = std.ArrayList(u8).init(allocator);
```

Comment:
Does this need an `errdefer list.deinit()`?
e.g. if the 2nd reallocation/growing fails?

Reply:
good call +1